docs/src/main/asciidoc/grpc-generation-reference.adoc

```diff
@@ -165,6 +165,15 @@ quarkus.generate-code.grpc.scan-for-proto-exclude."<groupId>:<artifactId>"=foo/p
 
 Note that `:` characters in the property keys must be escaped.
 
+== Kotlin code generation
+
+`protoc` also supports https://protobuf.dev/reference/kotlin/kotlin-generated/[generating Kotlin code] in addition to the generated Java code.
+
+By default, the Kotlin code generation is enabled if the dependency `io.quarkus:quarkus-kotlin` is present.
+You also have to include `com.google.protobuf:protobuf-kotlin` as a `compileOnly` dependency in your project.
```

Comment:
I would move that into an admonition mentioning it's only for gradle.

Reply:
Done